src/main/java/com/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -11,12 +12,15 @@
 @Data
 public class CreateBookRequestDto {
     @NotNull
+    @NotBlank
```

Comment:
You don't need both `@NotNull` and `@NotBlank` annotations. Please remove one of them and do it for all fields

Reply:
not agree. NotNull and NotBlank validate different things. Better use both of these annotations.